htdocs/core/class/commonobject.class.php

```diff
@@ -10378,6 +10378,15 @@ public function createCommon(User $user, $notrigger = 0)
 			}
 			$key_fields = $this->fields[$key];
 
+			// Check ref
+			if ($key == 'ref' && !empty($values['ref'])) {	// We check that ref is not already used
```

Comment:
Surely a bug here: We add a { and we add another one line 10384 but there is only one closing added.